AWS/terraform/main.tf

```diff
@@ -9,7 +9,7 @@ terraform {
 }
 
 provider "aws" {
-  profile = "default"
+  profile = "baradmin"
```

Comment:
I believe this may have been committed accidentally? I think we should keep this as default.

Reply:
Yes! Big catch. Thanks! 🙇